mergify_engine/config.py

```diff
@@ -158,6 +184,9 @@ def CommaSeparatedIntList(value: str) -> typing.List[int]:
 NOSUB_MAX_REPO_SIZE_KB: int
 GIT_EMAIL: str
 CONTEXT: str
+BOT_ACCOUNTS: typing.Dict[str, str]
```

Comment:
That should not be "bot account".
This is also going to be used for other things in the future, such as rebasing as the user for commands.